Gemfile.lock

```diff
@@ -182,64 +182,62 @@ GEM
       addressable (~> 2.7)
     letter_opener (1.7.0)
       launchy (~> 2.2)
-    listen (3.4.1)
+    listen (3.5.1)
```

Comment:
### [listen](https://github.com/guard/listen)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.4.1 → 3.5.1

[_[change-log](https://github.com/guard/listen/releases), [source-code](https://github.com/guard/listen/tree/v3.5.1)_]





<details>
<summary>Commits</summary>

A change of **8** commits. See the full changes on [the compare page](https://github.com/guard/listen/compare/v3.4.1...v3.5.1).

These are the individual commits:
- (8373721) [issue #533: suppress rubocop warning about size of gemspec b…](https://github.com/guard/listen/commit/8373721c8718568f80daf59df55b8f8c0bcc1b06)
- (21cb794) [issue #533: rescue only default StandardError](https://github.com/guard/listen/commit/21cb7944485f8f8029060cc61137c98e8c6cd539)
- (c24d860) [issue #533: debug with more specific :fake_worker names](https://github.com/guard/listen/commit/c24d8604adab6eb5a8085ed97214bc98dd350fe4)
- (57224cc) [issue #533: subject.stop so thread cannot keep running and l…](https://github.com/guard/listen/commit/57224cc4416b990b2361d53f39830cd32f545a7e)
- (133b1bf) [issue #533: fix indentation post PR review](https://github.com/guard/listen/commit/133b1bf90b124c5e7a80fae775f3d2cf1e89828d)
- (c00a57b) [release v3.5.0](https://github.com/guard/listen/commit/c00a57bffa057c520c5b7285420c21e3aff11454)
- (1243d92) [Bump required ruby version to v2.4.0](https://github.com/guard/listen/commit/1243d923e66e31ed28bfad755de6c12035b95720)
- (041fca5) [release v3.5.1](https://github.com/guard/listen/commit/041fca566926370963329232309b6cb4e59ebe9b)


</details>